public/blog/first-ever-truffle-retreat.md

```diff
@@ -0,0 +1,123 @@
+
+That just left Nick and I. Though we both knew Nick would work on the internals of Truffle when he was hired, we didn’t know the extent of the work required; in addition to improving Truffle, Nick would be burdened with the arduous task of softly and lovingly ripping my baby from my arms. That is, Truffle’s my baby, and I’ve been nurturing it from day one. In order to build a more functional and growing team, it became clear I’d have to let go of the internal implementation details and focus more on our business and our product, leaving Nick to explore the tool’s technical way forward.
+
+This transition has been a great growing experience already, only a few weeks after the retreat itself. We’ll continue to evaluate this structure over the next six months to see how it works out. 
+
+### Mission Statement
+
+The next step was to determine why we’re here. The big three questions were: Why are you here? What do you value? And what do you stand for? The point was to elicit each team member’s value systems to get a big picture of what it is we really care about. 
+
+My original plan for this session was to put each person on the spot and ask them those questions directly. As it turns out, this interrogation technique was not the best way to get meaningful, genuine answers from the team. 
+
+Luckily, John came up with a better plan. He suggested we have each member of the team write their values on six index cards, and then have the whole team talk about each other’s values, create a big list, and vote on them to determine our company values. 
+
+![Our values](/blog/images/first-ever-truffle-retreat/mission.jpg)
+
+The conversation helped us all understand how different we were, and how much our values varied. We expressed everything from receiving good pay to building the distributed ecosystem. 
+
+In the end, we all agreed why we’re here. Our new mission statement: 
+
+> _Truffle builds useful tools for the next generation of blockchain technologies. We enable the growth of the Ethereum and open source communities, help to improve the role of decentralization in society, and empower developers to achieve their goals._
+
+### Product Roadmap
+
+On Thursday, we got into the details of what we’ve been working on at Truffle and where we’re going. This included presentations for new products [SPOILER ALERT!] and discussions about specific features. We’ve been hard at work since we drafted this post, so some you may have heard of already: 
```

Comment:
Mentioning Truffle 4 and Ganache are fine, but do we really want to show our hand with things in our pipeline? I might suggest leaving off the things not released.

Reply:
Broad strokes are fine I think. For instance, see here: https://gnosis.pm/timeline

Gnosis has had 2018 planned out since the start of 2017. In general it's hard to tell if showing some of our cards increases motivation for people working on competitive products or decreases their motivation in that they can now respond with, "huh, I don't have to build that."

Additionally, teasing some things gets hype, but we don't *have* to stick to it, as well as keeping us in check, making sure we iterate fast.